src/doc/src/reference/config.md

```diff
@@ -36,7 +36,10 @@ it, named `/projects/foo/bar/baz/mylib` and `/projects/foo/bar/baz/mybin`, and
 there are Cargo configs at `/projects/foo/bar/baz/mylib/.cargo/config.toml`
 and `/projects/foo/bar/baz/mybin/.cargo/config.toml`, Cargo does not read
 those configuration files if it is invoked from the workspace root
-(`/projects/foo/bar/baz/`).
+(`/projects/foo/bar/baz/`), or anyways if it is asked to compile a crate from 
+that workspace, even if from outside the workspace root (e.g. via the 
```

Comment:
This sentence is already quite long. Would it be possible to keep the original period, and start this as a new sentence?